CHANGES.rst

```diff
@@ -76,6 +76,8 @@ Improvements
 - Add the ability to clone privacy settings (:pr:`6156`, thanks :user:`SegiNyn`)
 - Add option for managers to change the registration fee of a set of registrations (:issue:`6132`,
   :pr:`6138`)
+- Filter events by room in the calendar legend
+  (:issue:`6148, 6149`, :pr:`6158`, thanks :user:`Moliholy, unconventionaldotdev`)
```

Comment:
I'd edit it into this one:

> Add a legend to the category calendar, allowing to filter events either by category or location

-->

> Add a legend to the category calendar, allowing to filter events either by category, venue or room

---

Since it's part of the same release, having two separate changelog entries (and not even adjacent ones) is just confusing...

Reply:
Changed 👍